jujugui/static/gui/src/app/app.js

```diff
@@ -1097,9 +1097,12 @@ YUI.add('juju-gui', function(Y) {
       // Auto place the units. This is probably not the best UX, but is required
       // to display the machines in the deployment flow.
       this._autoPlaceUnits();
-      let cloud = env.get('cloud');
+      let cloud = env.get('providerType');
```

Comment:
The env only has `providerType`, I couldn't see anything about a `cloudType`, not sure if that's a bug.